recipes/discasm/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.1.2" %}
+
+package:
+    name: discasm
+    version: {{ version }}
+
+source:
+  fn: DISCASM-v{{ version }}.tar.gz # [unix]
+  url: https://github.com/DISCASM/DISCASM/releases/download/DISCASM-v{{ version }}/DISCASM-v{{ version }}.tar.gz # [unix]
+  md5: 571b2265018608298752f40f3f375b0f # [unix]
+
+build:
+    number: 0
+    skip: true  # [osx or py3k]
+    # The reason for skipping osx is that trinity >=2.4 is not available in the osx channel
+    # The python code in DISCASM runs on python 2.7.
```

Comment:
Thanks for adding those comments! IMO, those are tremendously helpful!

Reply:
@mbargull you're welcome. Also, changed build.sh to incorporate concern about moving everything to bin. I move it to share, as suggested by @daler, but rather than creating a symlink, I create a script that invokes DISCASM using its full path. This is the method used by the star-fusion tool, for reasons similar to the situation with this tool.

Reply:
Right, the script is the way to go since with a symlink would result in `__file__` (inside your Python script) being the path of the link not the destination -- I hadn't thought of that before, good catch!

Reply:
[BTW: (unspoken) convention is to use 2 spaces instead of 4 for the indentation inside the `meta.yaml` -- but that's just nitpicking ;)]